src/ExtendedXmlSerializer/IExtendedXmlSerializerConfig.cs

```diff
@@ -27,12 +27,14 @@ namespace ExtendedXmlSerialization
 {
     public interface IExtendedXmlSerializerConfig
     {
-        Type Type { get; }
-        int Version { get; set; }
+        Type Type { get; } // Consider making obsolete as logic is now contained within implementation.
+        int Version { get; set; } // Consider making getter only, defined by implementation.
```

Comment:
Haha well alright.  I was just making sure.  I will the setter and checkin the commit.